diff --git a/dist/modules/auth/lib/backendChrome.js b/dist/modules/auth/lib/backendChrome.js
index 8a0d504d687f83dd63a9e3e0fd60e5e098faf8e3..d685a012bbe253059cf5735363f613225bd74079 100644
--- a/dist/modules/auth/lib/backendChrome.js
+++ b/dist/modules/auth/lib/backendChrome.js
@@ -71,6 +71,7 @@ function normalizeGroupWeights(groups) {
     "customers.nav.group",
     "catalog.nav.group",
     "customers~sales.nav.group",
+    "customers~sales~cpq.nav.group",
     "resources.nav.group",
     "staff.nav.group",
     "entities.nav.group",
diff --git a/src/modules/auth/lib/backendChrome.tsx b/src/modules/auth/lib/backendChrome.tsx
index 278009549856685827bdbb79bcaf050ede68334f..db1eda020f5f7652c2f53fcb965f11f994ac9fcc 100644
--- a/src/modules/auth/lib/backendChrome.tsx
+++ b/src/modules/auth/lib/backendChrome.tsx
@@ -145,6 +145,7 @@ function normalizeGroupWeights(groups: NavGroupWithWeight[]): NavGroupWithWeight
     'customers.nav.group',
     'catalog.nav.group',
     'customers~sales.nav.group',
+    'customers~sales~cpq.nav.group',
     'resources.nav.group',
     'staff.nav.group',
     'entities.nav.group',
